packages/@webex/plugin-meetings/src/meeting/util.ts

```diff
@@ -124,6 +125,7 @@ const MeetingUtil = {
         locale: options.locale,
         deviceCapabilities: options.deviceCapabilities,
         liveAnnotationSupported: options.liveAnnotationSupported,
+        ipVersion: meeting.getWebexObject().meetings.reachability.getIpVersion(),
```

Comment:
question: Did you need to do getWebexObject again here? Could you not have used the `webex` from line 98?

Reply:
I introduced this `getWebexObject`, but i don't remember why ...

Reply:
in some places in SDK we're doing `.webex` and some others `.getWebexObject()` - I've just used the same approach to whatever was done in the surrounding code in the same file to avoid having to change the mocks in the unit tests for that file